docs/node/run-your-node/sentry-node.md

```diff
@@ -225,78 +188,44 @@ Before using this configuration you should collect the following information to
   on the sentry node.
 
 ```yaml
-##
-# Oasis Node Configuration
-#
-# This file's configurations are derived from the command line args found in
-# the root command of the oasis-node binary. For more information execute:
-#
-#     oasis-node --help
-#
-# Settings on the command line that are separated by a dot all belong to the
-# same nested object. So "--foo.bar.baz hello" would translate to:
-#
-#     foo:
-#       bar:
-#         baz: hello
-##
-
-# Set this to where you wish to store node data. The node artifacts
-# should also be located in this directory (for us this is /node/data)
-datadir: /node/data
-
-# Logging.
-#
-# Per-module log levels are defined below. If you prefer just one unified log
-# level, you can use:
-#
-# log:
-#   level: debug
-log:
-  level:
-    # Per-module log levels. Longest prefix match will be taken. Fallback to
-    # "default", if no match.
-    default: debug
-    tendermint: warn
-    tendermint/context: error
-  format: JSON
-  # By default logs are output to stdout. If you're running this in docker keep
-  # the default
-  #file: /var/log/oasis-node.log
-
-# Path to the genesis file for the current version of the network.
+mode: validator
+common:
+    data_dir: /node/data
+    log:
+        format: JSON
+        level:
+            cometbft: warn
+            cometbft/context: error
+            # Per-module log levels. Longest prefix match will be taken.
+            # Fallback to "default", if no match.
+            default: debug
+        # By default logs are output to stdout. If you're running this in docker keep
+        # the default
+        #file: /var/log/oasis-node.log
+
+consensus:
+    listen_address: tcp://0.0.0.0:26656
+    p2p:
+        disable_peer_exchange: true
+        persistent_peers:
+            - {{ sentry_node_tendermint_id }}@{{ sentry_node_private_ip }}:26656
+    prune:
```

Comment:
Does anyone know why did we have pruning configured here for the validator? e.g. for normal validator node we don't have pruning configured in the examples. I would just remove this "prune section" from this config (i think it's ok to keep in the other config above, it makes more sense there).